src/app/support/services/support.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { Observable } from 'rxjs';
+
+import { ISupport } from '../interfaces/support.interface';
+
+@Injectable({
+  providedIn: 'root',
+})
+
+export class SupportService {
+
+  private readonly _supportUrl = 'api/support/';
+
+  constructor(private _httpClient: HttpClient) { }
+
+  public send(message: ISupport): Observable<object> {
```

Comment:
I suppose we need to create base api service which will contain injected `HttpClient` and abstract `_url` variable. Other service will be inherited from the base service. It allows to avoid duplicate code and unnecessary dependencies.